### PR TITLE
BAH-1720|configure-the-patient-queues-active-and-all-to-search-the-pa…

### PR DESCRIPTION
**Pull request by shilpa-iplit**
BAH-1720|configure-the-patient-queues-active-and-all-to-search-the-patients-by-abha-id

### CODE REVIEW

**Comment by gsluthra** [comment]
Can we add Tests? Maybe in BahmniPatientDaoIT or BahmniPatientDaoImplIT or a new test class?

File: bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/BahmniPatientDaoImplLuceneIT.java [hunk 1]
```diff
@@ -222,4 +222,63 @@ public void shouldNotReturnDuplicatePatientsEvenIfTwoIdentifiersMatches() {
         assertTrue(patient.getExtraIdentifiers().contains("200006"));
     }
 
+    @Test
+    public void shouldSearchByPatientPrimaryIdentifierThenByName1() {
```
[the remark this answers is not in the document]
Comment (shilpa-iplit) [reply bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/BahmniPatientDaoImplLuceneIT.java]: Addressed review comments. Please check.  @gsluthra

**Comment by gsluthra** [comment]
Seems fine to me now. Will request @N0-man or @angshu to also review. If one of them approves, then we are good to merge. thanks!

**Comment by gsluthra** [comment]
Ok from my side.

**Review by N0-man — APPROVED** [review]

**Pull request merged**